glean-core/src/internal_pings.rs

```diff
@@ -21,53 +21,54 @@ pub struct InternalPings {
 impl InternalPings {
     pub fn new(enabled: bool) -> InternalPings {
         InternalPings {
-            baseline: PingType::new_internal(
+            baseline: PingType::new(
                 "baseline",
                 true,
                 true,
                 true,
                 true,
+                enabled,
                 vec![
                     "active".to_string(),
                     "dirty_startup".to_string(),
                     "inactive".to_string(),
                 ],
-                enabled,
             ),
-            metrics: PingType::new_internal(
+            metrics: PingType::new(
                 "metrics",
                 true,
                 false,
                 true,
                 true,
+                enabled,
                 vec![
                     "overdue".to_string(),
                     "reschedule".to_string(),
                     "today".to_string(),
                     "tomorrow".to_string(),
                     "upgrade".to_string(),
                 ],
-                enabled,
             ),
-            events: PingType::new_internal(
+            events: PingType::new(
                 "events",
                 true,
                 false,
                 true,
                 true,
+                enabled,
                 vec![
                     "startup".to_string(),
                     "inactive".to_string(),
                     "max_capacity".to_string(),
                 ],
-                enabled,
             ),
             deletion_request: PingType::new(
                 "deletion-request",
                 true,
                 true,
                 true,
                 true,
+                true, // The deletion-request should not be disabled
```

Comment:
Hrm, where else can we catch this... Guess we can't put this logic in Experimenter where it'd be most helpful?

Reply:
We can incorporate a validation for this in the Server Knobs "configurator" tooling that is coming eventually. I'll add a note to the relevant bug.